src/app/configuration.ts

```diff
@@ -28,8 +28,8 @@ export class Configuration {
      * Select the correct content-type to use for a request.
      * Uses {@link Configuration#isJsonMime} to determine the correct content-type.
      * If no content type is found return the first found type if the contentTypes is not empty
-     * @param {string[]} contentTypes - the array of content types that are available for selection
-     * @returns {string} the selected content-type or <code>undefined</code> if no selection could be made.
+     * @param contentTypes - the array of content types that are available for selection
```

Comment:
Did swagger-codegen update their generator? Did they fix the Observable import?

Reply:
yeah looks like they did